lib/api/controllers/documentController.js

```diff
@@ -49,7 +49,7 @@ function DocumentController (kuzzle) {
    * @returns {Promise<Object>}
    */
   this.scroll = function documentScroll (request) {
-    if (!request.input.args.scroll) {
+    if (!request.input.body || !request.input.body.scroll) {
       throw new BadRequestError('document:scroll must specify an argument "scroll"');
     }
     if (!request.input.args.scrollId) {
```

Comment:
Or to follow the action's documentation for HTTP protocol, this should be 
`request.input.resource.scrollId`

and we should fix the API documentation for other protocols with following:
```javascript
{
  "controller": "document",
  "action": "scroll",
  "scrollId": "<scrollId>"
  "body": {
    // The scroll parameter tells Elasticsearch to keep the search context open for another 1m.
    "scroll": "1m"
  }
}

```